code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -244,12 +246,14 @@
 /mob/living/simple_animal/hostile/guardian/punch/AttackingTarget()
 	..()
 	if(istype(target, /mob/living))
-		src.say("[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]\
-		[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]")
+		if(src.cry_wait == 0)
```

Comment:
Weirdness would happen like punching one guy twice, then minutes later punching another and only having the battlecry come on the third punch.